pkg/grpc/universalapi/api_metadata.go

```diff
@@ -78,12 +100,13 @@ func (a *UniversalAPI) GetMetadata(ctx context.Context, in *emptypb.Empty) (*run
 	}
 
 	return &runtimev1pb.GetMetadataResponse{
-		Id:                   a.AppID,
-		ExtendedMetadata:     extendedMetadata,
-		RegisteredComponents: registeredComponents,
-		ActiveActorsCount:    activeActorsCount,
-		Subscriptions:        ps,
-		HttpEndpoints:        registeredHTTPEndpoints,
+		Id:                      a.AppID,
+		ExtendedMetadata:        extendedMetadata,
```

Comment:
Should we put the Dapr version in its own key? Right now it's inside `extendedMetadata` and that seems odd.

(I'm not saying to remove it from `extendedMetadata` since that would be backwards-incompatible, but to ALSO add it to its own key)

Reply:
I agree with that. Created this issue - https://github.com/dapr/dapr/issues/6528